js/exporter/jspdf/v3/export_data_grid_3.js

```diff
@@ -43,22 +44,36 @@ function exportDataGrid(doc, dataGrid, options) {
 
             initializeCellsWidth(rowsInfo, options.columnWidths); // customize via options.colWidths only
 
-            // TODO set/update/initColSpanRowSpan(rows);
+            // apply colSpans + recalculate cellsWidth
+            // TODO: applyColSpans();
 
             // set/update/initCellHeight - autocalculate by text+width+wordWrapEnabled or use value from customizeCell
             calculateHeights(doc, rowsInfo, options);
 
-            // TODO set/update/initBorders(rows);
+            // apply rowSpans + recalculate cells height
+            // TODO: applyRowSpans();
 
+            // when we known all sizes we can calculate all coordinates
             calculateCoordinates(doc, rowsInfo, options); // set/init/update 'pdfCell.top/left'
 
+            // recalculate for grouped rows
+            // TODO: applyGroupIndents()
+
+            // set/update/initBorders(rows);
+            // TODO: initBorders(rows);
+
+            // splitting to pages
+            // TODO: applySplitting()
+            // ??? maybe after [][] => [], and cellInfo must contain rowIndex + colIndex ( for sorting, filtering )
```

Comment:
Here, we can 'split' a cell with 'colSpan > 0' (for example a long 'band' cell) into two cells and place the first cell on the first page and second cell on the second page. Most of 'split rectangles by pages' algorithms will not split rectangles.
And, we can show initial 'text' in the both new cells.
The same approach we can apply to cells with 'rowSpan > 0' (row headers in Pivot).
Let's add this comment into PR.